atest/testdata/standard_libraries/builtin/set_test_message.robot

```diff
@@ -21,6 +21,23 @@ Set Multiline Message
     Set Test Message    1\n2\n3
     [Teardown]    Should Be Equal    ${TEST MESSAGE}    1\n2\n3
 
+Use the HTML
+    [Documentation]    PASS *HTML* custom message and another custom message in <HR> with a line
+    Set Test Message  custom message
+	Set Test Message  *HTML* and another custom message in <HR> with a line  append=True
```

Comment:
There seems to be a tab on the line above. More tabs on some of the lines below. They should all be converted to spaces.